src/servers/ZoneServer2016/entities/constructiondoor.ts

```diff
@@ -55,18 +55,39 @@ function getMaxHealth(itemDefinitionId: Items): number {
 }
 
 export class ConstructionDoor extends DoorEntity {
+  /** CharacterId of the player that owns the door entity */
   ownerCharacterId: string;
+
+  /** Hashed password for the door */
   passwordHash: number = 0;
+
+  /** List of all accessed players on a door entity: demolish permission also grants access */
   grantedAccess: Array<string> = [];
 
+  /** The parent object the door entity is attached to */
   parentObjectCharacterId: string;
+
+  /** ItemDefinitionId of the door */
   readonly itemDefinitionId: number;
+
+  /** Current slot from the parent object the door is attached to */
   readonly slot: string;
+
+  /** Range at which the door entity will receive damage from explosions */
   damageRange: number;
+
+  /** Fixed position the door remains at whilst closed */
```

Comment:
```suggestion
  /** Fixed position the door remains at while it’s closed */
```

Reply:
hahaha that's just a 🤓🤓 english word, whilst = while it's

Reply:
ahah never saw it